src/ethereum/forks/amsterdam/vm/instructions/system.py

```diff
@@ -110,6 +110,9 @@ def generic_create(
 
     evm.accessed_addresses.add(contract_address)
 
+    # Track address access for BAL
+    track_address_access(state.change_tracker, contract_address)
```

Comment:
Random comment, not a serious review, but if this is an operation on the "state", then should `track_address_access` only take the state itself as the first argument? Alternatively, if this is an operation on the "state tracker", then does it make sense for the tracker to be a property of `State` (or should it be a property of, for example, `BlockEnvironment`)?